services.yaml

```diff
@@ -104,6 +225,23 @@ paths:
           description: TBD
         401:
           $ref: "#/definitions/UnauthorizedError"
+  /services/{service_id}/deployments:
```

Comment:
what about servers/agents lists?

Reply:
added with disclaimer (not yet implemented)